tyr/servers/server.py

```diff
@@ -371,21 +386,58 @@ def blockdevicemapping(self):
 
         return bdm
 
+    def get_subnet_vpc_id(self, subnet_id):
+        vpcConn = VPCConnection()
+        subnets = vpcConn.get_all_subnets(
+            filters={'subnet_id': subnet_id})
+        if len(subnets) == 1:
+            vpc_id = subnets[0].vpc_id
+            return vpc_id
+        else:
+            self.log.error("None or greater then 1 subnet returned")
+            exit(1)  # should probably throw an exception, if I knew how
```

Comment:
To raise a generic exception it would just be

```python
raise Exception("Something went wrong")
```

Alternatively you could add a new exception type to `tyr/servers/exceptions.py` and raise that.

``` python
# tyr/servers/exceptions.py

class NoSubnetReturned(Exception):
    pass

# tyr/servers/server.py

raise NoSubnetReturned("No subnet was returned")
```